web/src/pages/network/NetworkGateway/NetworkGateway.tsx

```diff
@@ -76,6 +133,25 @@ export const NetworkGatewaySetup = () => {
           </ExpandableCard>
         </>
       )}
+      {/* From Package */}
+      <h3>{LL.gatewaySetup.header.fromPackage()}</h3>
+      <MessageBox>
+        {parse(
+          LL.gatewaySetup.messages.fromPackage({
+            setupGatewayDocs: externalLink.gitbook.setup.gateway,
+          }),
+        )}
+      </MessageBox>
+      <Button
```

Comment:
Just wrap this button in `<a>` tag with target of "_self"

Reply:
For now unnecessary, maybe in the future this could be available